haiku-apps/optipngtranslator/optipngtranslator-0.0.1.recipe

```diff
@@ -3,11 +3,15 @@ DESCRIPTION="
 The OptiPNGTranslator for Haiku allows you to save images as PNG files which \
 have been optimized by OptiPNG.
 "
-HOMEPAGE="https://github.com/noryb009/OptiPNGTranslator/"
+HOMEPAGE="https://github.com/noryb009/OptiPNGTranslator"
 SOURCE_URI="git://github.com/noryb009/OptiPNGTranslator.git#8e3996a92d840f3c097cf954e001d4f371265161"
 LICENSE="MIT"
 COPYRIGHT="2013 Luke (noryb009)"
-REVISION="1"
+REVISION="2"
+COMMIT="8e3996a92d840f3c097cf954e001d4f371265161"
+SOURCE_URI="$HOMEPAGE/archive/$COMMIT.tar.gz"
+CHECKSUM_SHA256="4bbcb970f42bc13bba1105cce03739601339c3c52f8aec6c16130a9a1834746d"
+SOURCE_DIR="optipngtranslator-$COMMIT"
```

Comment:
I would replace
```
REVISION="2"
COMMIT="8e3996a92d840f3c097cf954e001d4f371265161"
SOURCE_URI="$HOMEPAGE/archive/$COMMIT.tar.gz"
CHECKSUM_SHA256="4bbcb970f42bc13bba1105cce03739601339c3c52f8aec6c16130a9a1834746d"
SOURCE_DIR="optipngtranslator-$COMMIT"
```
by
```
REVISION="1"
srcGitRev="8e3996a92d840f3c097cf954e001d4f371265161"
SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
CHECKSUM_SHA256="4bbcb970f42bc13bba1105cce03739601339c3c52f8aec6c16130a9a1834746d"
SOURCE_DIR="optipngtranslator-$srcGitRev"
```
That's because srcGitRev is imho a slightly better choice than COMMIT, since:
* there are already 36 recipes using "srcGitRev", compared to only 4 which are using "COMMIT";
* contrary to "COMMIT", which looks like a reserved HaikuPorter variable, "srcGitRev" makes it clear that it is not a reserved but a custom variable.

And since the resulting SOURCE_URI is unchanged we don't really need to bump REVISION.

But if another contributor says that COMMIT is ok and/or that we should bump REVISION then that's OK for me too ;-)

Hmm, can you please add a blank line just before ARCHITECTURES?